spec/index.bs

```diff
@@ -1337,6 +1341,33 @@ enum GPUPowerPreference {
 {{GPURequestAdapterOptions}} has the following members:
 
 <dl dfn-type=dict-member dfn-for=GPURequestAdapterOptions>
+    : <dfn>allowSoftware</dfn>
+    ::
+        Expresses a restriction on whether "software" implementations should be returned by an
+        adapter request.
+
+        This behavior is implementation-defined: user agents may not always obey this option.
+        For example, if a system has very limited GPU hardware but a fast software implementation,
+        the user agent may always prefer the software implementation.
+
+        It must be one of the following <dfn enum for=>GPUAllowSoftwareOption</dfn> values:
+
+        <dl dfn-type=enum-value dfn-for=GPUAllowSoftwareOption>
+            : <dfn>"never"</dfn>
+            ::
+                Adapters with major performance caveats should never be allowed.
```

Comment:
still talks about the major performance caveats...

Reply:
oops.